otel/conf.go

```diff
@@ -421,11 +421,16 @@ windowsperfcounters/mssql_{{.MSSQLID}}:
         action: update
         new_name: processes/cpu_time
         operations:
+          - action: experimental_scale_value
+            experimental_scale: 1000000
           # change data type from double -> int64
           - action: toggle_scalar_data_type
           - action: add_label
             new_label: process
             new_value: all
+          - action: delete_label_value
```

Comment:
Does this need a comment? E.g., "retain only user and syst state label values"?

Reply:
done